File: src/main/java/com/wikia/tibia/repositories/OutfitRepository.java
package com.wikia.tibia.repositories;

import com.wikia.tibia.gateways.OutfitGateway;
import com.wikia.tibia.objects.Outfit;

public class OutfitRepository extends WikiObjectRepository {

    public OutfitRepository() {
        super(Outfit.class, new OutfitGateway());
    }
}

File: src/main/kotlin/com/wikia/tibia/repositories/OutfitRepository.kt
package com.wikia.tibia.repositories

import com.wikia.tibia.enums.Contract
import com.wikia.tibia.gateways.WikiObjectGateway
import com.wikia.tibia.objects.Outfit

class OutfitRepository : WikiObjectRepository<Outfit>(
    wikiObjectClass = Outfit::class.java,
    wikiObjectGateway = WikiObjectGateway(Contract.OUTFITS)
)
